api_app/api/routes/api.py

```diff
@@ -14,12 +14,15 @@
     {"name": "status", "description": "Status of API and related resources"},
 ]
 
+
 router = APIRouter()
-router.include_router(health.router, tags=["health"])
-router.include_router(workspace_templates.router, tags=["workspace templates"])
-router.include_router(workspace_service_templates.service_templates_router, tags=["workspace service templates"])
-router.include_router(workspace_service_templates.user_resource_templates_router, tags=["user resource templates"])
-router.include_router(workspaces.workspaces_router, tags=["workspaces"])
-router.include_router(workspaces.workspace_services_router, tags=["workspace services"])
-router.include_router(workspaces.user_resources_router, tags=["user resources"])
-router.include_router(status.router, tags=["status"])
+core_router = APIRouter(prefix=config.API_PREFIX)
+core_router.include_router(health.router, tags=["health"])
+core_router.include_router(workspace_templates.router, tags=["workspace templates"])
+core_router.include_router(workspace_service_templates.workspace_service_templates_router, tags=["workspace service templates"])
+core_router.include_router(workspace_service_templates.user_resource_templates_router, tags=["user resource templates"])
+core_router.include_router(status.router, tags=["status"])
+core_router.include_router(workspaces.tre_router, tags=["workspaces"])
+
+router.include_router(core_router)
+router.include_router(workspaces.workspace_router)
```

Comment:
do you also need the 'tag=["workspace"]'?

Reply:
is it on purpose that the user resources router got deleted?

Reply:
Its part of workspaces.workspace_router

the tags are in workspaces.py, maybe we can rename/move things around as is a bit confusing. The worksapces.py has too much in and needs to be split IMHO. But didn't want to chnge too many files.

Reply:
I've updated the tags, and reorganized the routers, does this look correct?

<img width="1097" alt="image" src="https://user-images.githubusercontent.com/17089773/141823595-513cc59d-496f-49a4-9d21-394809ad8828.png">

Reply:
looks good